go.mod

```diff
@@ -27,7 +27,7 @@ require (
 )
 
 require (
-	github.com/fastly/go-fastly/v9 v9.3.2
+	github.com/fastly/go-fastly/v9 v9.4.1-0.20240528142352-98043629c62b
```

Comment:
This is the current `main` branch as of today. We probably want to release actual 9.4.1 (or 9.5)

Reply:
Is this needed for this PR?

Reply:
I've just published https://github.com/fastly/go-fastly/releases/tag/v9.5.0 so let's update this dependency reference